src/network/network.h

```diff
@@ -266,6 +271,9 @@ class NetworkAddress
 	void Resolve(const char* host, unsigned short port, bool nonblocking = true);
 	int GetResolveStatus(void);
 
+	const char* getRawHost();
```

Comment:
This was an change I made earlier for testing that accidentally made it into this. By retrieving the host from the connection object, you could restart the connection process. This isn't used anywhere now, though.